src/AzSK.ADO/Framework/Abstracts/ADOSVTBase.ps1

```diff
@@ -460,10 +461,24 @@ class ADOSVTBase: SVTBase {
 			#added check azuretable check here, if ((azuretable is used for storing bug info and scan mode is CA) OR azuretable bug info is disabed) then only allow bug logging
 			$scanSource = [AzSKSettings]::GetInstance().GetScanSource();
 			$isAzureTableEnabled = [Helpers]::CheckMember($this.ControlSettings.BugLogging, "UseAzureStorageAccount");
+			
+			#if bug logging is enabled for inactive resources, then only bug will be logged for inactive resources.
+			if([Helpers]::CheckMember($this.ControlSettings.BugLogging, "BugLogForInactiveResources"))
+			{
+				$bugLogForInactiveResources = $true;
+			}
+			else {
+				$bugLogForInactiveResources = $this.isResourceActive;
+			}
 			if (!$isAzureTableEnabled -or ($isAzureTableEnabled -and ($scanSource -eq "CA")) )
 			{
-				if (($ControlResults.ControlResults.VerificationResult -contains "Failed") -or ($ControlResults.ControlResults.VerificationResult -contains "Verify")) {
-					$this.BugLoggingPostEvaluation($ControlResults, $BugLogParameterValue)
+				if ($bugLogForInactiveResources) {
+					if (($ControlResults.ControlResults.VerificationResult -contains "Failed") -or ($ControlResults.ControlResults.VerificationResult -contains "Verify")) {
+						$this.BugLoggingPostEvaluation($ControlResults, $BugLogParameterValue)
+					}
+				}
+				else {
+					$this.PublishCustomMessage("Bug logging is disabled for inactive resources.", [MessageType]::Info);
```

Comment:
The current resource is inactive. Bug logging is disabled for inactive resources.

Reply:
Warning

Reply:
changes done